contrib/platform/src/com/sun/jna/platform/win32/COM/IAccessible.java

```diff
@@ -0,0 +1,125 @@
+/* Copyright (c) 2021 Mo Beigi, All Rights Reserved
+ *
+ * The contents of this file is dual-licensed under 2
+ * alternative Open Source/Free licenses: LGPL 2.1 or later and
+ * Apache License 2.0. (starting with JNA version 4.0.0).
+ *
+ * You can freely decide which license you want to apply to
+ * the project.
+ *
+ * You may obtain a copy of the LGPL License at:
+ *
+ * http://www.gnu.org/licenses/licenses.html
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "LGPL2.1".
+ *
+ * You may obtain a copy of the Apache License at:
+ *
+ * http://www.apache.org/licenses/
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "AL2.0".
+ */
+package com.sun.jna.platform.win32.COM;
+
+import com.sun.jna.platform.win32.Guid.IID;
+import com.sun.jna.platform.win32.Variant.VARIANT;
+import com.sun.jna.platform.win32.WTypes.BSTRByReference;
+import com.sun.jna.platform.win32.WinNT.HRESULT;
+import com.sun.jna.ptr.IntByReference;
+
+/**
+ * IAccessible Interface
+ *
+ * Exposes methods and properties that make a user interface element and its children accessible to client applications.
+ * The IAccessible interface inherits from the IDispatch interface.
+ *
+ * @see <a href="https://docs.microsoft.com/en-us/windows/win32/api/oleacc/nn-oleacc-iaccessible">IAccessible interface (oleacc.h)</a>
+ * @author Mo Beigi, me@mobeigi.com
+ */
+public interface IAccessible extends IDispatch
```

Comment:
Is there a real good reason for having `IAccessible` and `Accessible`? 

@dbwiddis I have yet to see a good reason for having the interface. Do you see one? I don't see the option to have a different option, as calling into COM will always require the implementation to yield a pointer as "this" and thus a `PointerType` is the most appropriate base type.

Reply:
The best reason is to avoid the confusion and inconsistency in existing JNA mappings with the “I” prefix.  :-)

Most/all things can be done without the interface, but it would be good to actually state a standard we can try to stick with. 

Traveling and poor internet so won’t be looking at this for a week more days.